packages/ace-report/src/l10n/locales/fr.json

```diff
@@ -0,0 +1,72 @@
+{
+    "bestpractice": "Bonnes Pratiques",
```

Comment:
Par cohérence avec l'anglais, peut–être qu'un singulier serait plus approprié?

-> "Bonne pratique"

Reply:
Ah, I made the same mistake in GUI! :(

Ace FR:

https://github.com/daisy/ace/blob/0b94c2e04c45b7f31fc98f02666c47218f56ba4b/packages/ace-report/src/l10n/locales/fr.json#L2

Ace EN:

https://github.com/daisy/ace/blob/0b94c2e04c45b7f31fc98f02666c47218f56ba4b/packages/ace-report/src/l10n/locales/en.json#L2

Ace GUI FR:

https://github.com/daisy/ace-gui/blob/191da98a59614460be8a82dd6ac21c107f2f64f1/src/shared/l10n/locales/fr.json#L106

Ace GUI EN:

https://github.com/daisy/ace-gui/blob/191da98a59614460be8a82dd6ac21c107f2f64f1/src/shared/l10n/locales/en.json#L106

Reply:
https://github.com/daisy/ace-gui/commit/1a0113f

Reply:
https://github.com/daisy/ace/commit/3a94c27